QTMSettings.cs

```diff
@@ -95,8 +129,21 @@ public class SettingsBone : SettingsBase
     [XmlRoot("The_3D")]
     public class Settings3D : SettingsBase
     {
-        [XmlElement("AxisUpwards")]
+        [XmlIgnore]
         public Axis AxisUpwards;
+        [XmlElement("AxisUpwards")]
+        public string ModelAsString
```

Comment:
Does the XML serializer work on private members? If so, all of these should preferably be made private to avoid users setting them directly.

Reply:
No only with public. It would be cleaner interface if the xyAsString methods could be made private.